src/myrtlespeech/model/hard_lstm.py

```diff
@@ -0,0 +1,561 @@
+"""This implementation of a hard LSTM can be exported to ONNX and run on
+ONNX Runtime (ORT). In order to capture the control flow (e.g. looping over
+timesteps), it is necessary to :py:meth`torch.jit.script` the module before
+:py:meth`torch.onnx.export`-ing it. In other words, the :py:class:`HardLSTM`
```

Comment:
Missing colon stopping Sphinx rendering it correctly:

```suggestion
:py:meth:`torch.onnx.export`-ing it. In other words, the :py:class:`HardLSTM`
```

Reply:
wow good spot :sunglasses: